specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -3845,8 +3849,7 @@
           "type": "string",
           "description": "Enumerates the possible values for the encoding format of capture description.",
           "enum": [
-            "Avro",
-            "AvroDeflate"
```

Comment:
Deleting an enum value is a Binary Breaking Change in SDKs; this is typically not permitted.